src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -14,8 +14,10 @@ namespace Microsoft.CodeAnalysis.CSharp
     internal static partial class BoundExpressionExtensions
     {
         /// <summary>
-        /// Returns the RefKind if the expression represents a symbol
-        /// that has a RefKind, or RefKind.None otherwise.
+        /// Returns the RefKind of an expression
+        /// if the expression represents a symbol, the RefKind of the symbol is returned
+        /// if the expression is a "by-ref" expression like conditional operator or InlineArrayAccess the RefKind is calculated and returned
```

Comment:
>if the expression is a "by-ref" expression like conditional operator or InlineArrayAccess the RefKind is calculated and returned [](http://example.com/codeflow?start=12&length=128)

I am somewhat uncomfortable with this change. This helper is used in several places. It hasn't been demonstrated that any of the existing call sites are actually going to benefit from this change. Moreover, there is a risk that some of the existing call sites could be negatively affected by this change. And that is the risk that we probably don't want to take that late in the release cycle, especially without a proven benefit. I suggest to either demonstrating a benefit for an existing call site, or take a more conservative approach by specially handling ```BoundConditionalOperator``` only in the place where we intend to change the behavior.

Reply:
@bernd5 Was this feedback addressed?